### PR TITLE
Fix command install for helloworld

### PR DESCRIPTION
**Pull request by jsancio**
Was able to install against DCOS.

### CODE REVIEW

File: repo/meta/schema/package-schema.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
       "description": "True if this package installs a new Mesos framework."
     },
     "postInstallNotes": {
-      "type": "string"
+      "type": "string",
+      "description": "Post installation notes that would be useful to the user of this packages."
```
Comment (mgummelt) [inline repo/meta/schema/package-schema.json]: make package singular

File: repo/packages/H/helloworld/0/command.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "pip": [
-    "http://downloads.mesosphere.io/dcos-cli/dcos-0.1.0-py2.py3-none-any.whl",
+    "dcos",
```
Comment (mgummelt) [inline repo/packages/H/helloworld/0/command.json]: probably want to pin major number (<1.0)

**Comment by mgummelt** [comment]
LGTM after comments are addressed :penguin:

File: repo/meta/schema/package-schema.json (hunk 1 above)
Comment (jsancio) [inline repo/meta/schema/package-schema.json]: Done.

File: repo/packages/H/helloworld/0/command.json (hunk 2 above)
Comment (jsancio) [inline repo/packages/H/helloworld/0/command.json]: Done.

**Comment by jsancio** [comment]
Thanks! Will merge once TC is green.

**Pull request merged**